pkg/apply/testdata/apply/double_release/microservice-one.yaml

```diff
@@ -1,20 +1,23 @@
-apiVersion: helm.fluxcd.io/v1
+apiVersion: helm.toolkit.fluxcd.io/v2alpha1
 kind: HelmRelease
 metadata:
   name: microservice
   namespace: simple
-  annotations:
-    fluxcd.io/automated: "false"
 spec:
+  interval: 1m0s
+  chart:
+    spec:
+      chart: single-test
+      sourceRef:
+        kind: HelmRepository
+        name: microservice
+        namespace: simple
+      version: '>4.0.0'
```

Comment:
The `preHookX` and `testHookX` values defined below are only meaningful to the chart we defined as part of this project AFAIK, but if the referenced `microservice` chart provides those capabilities we don't need to preserve our own versions.  We needed a means to force the Helm release to fail during the pre-release and testing phases at the time - maybe that's not necessary anymore.

Reply:
I just converted to helm-controller format, lets change this in a separate patch